pkg/config/agent/config.go

```diff
@@ -185,6 +185,9 @@ type AgentConfig struct {
 	// The names of the interfaces on Nodes that are used to forward multicast traffic.
 	// Defaults to transport interface if not set.
 	MulticastInterfaces []string `yaml:"multicastInterfaces,omitempty"`
+	// The interval for antrea-agent to send IGMP query to pods, time uint is second.
+	// Defaults to 125 seconds.
+	IGMPQueryInterval int `yaml:"igmpQueryInterval"`
```

Comment:
@wenyingd @liu4480 do we likely have other similar configurations in the future? I wonder if we should group them in one struct, including multicastInterfaces.
cc @antoninbas

Reply:
I can't say no other configurations are needed in future. Agree to use one struct to group them if possible.

Reply:
OK, I will add a group for multicast

Reply:
grouping them sounds good to me.

Reply:
done